forumMessages/views.py

```diff
@@ -4,15 +4,20 @@
 
 
 def ViewMessages(request):
+    print("here")
```

Comment:
Unnecessary

Reply:
You're right, it's a redundant line that we used to debug some problems.

Reply:
Fixed it.